src/App.js

```diff
@@ -10,30 +10,37 @@ class App extends Component {
         super();
 
         this.state = {
-            data: null
+            inputData: [],
+            inputValue: ''
```

Comment:
It's better to store inputValue inside InputComponent and send it to App component as a callback function argument.